packages/components/src/components/PackageUpload.tsx

```diff
@@ -139,9 +139,15 @@ const PackageUploadForm: React.FC<PackageUploadFormProps> = ({
                       defaultValue={null}
                       control={control}
                     />
-                    {errors.communities?.message && (
-                      <FormErrorMessage>{errors.communities.message}</FormErrorMessage>
-                    )}
+                    {errors.communities &&
+                      errors.communities.map(
```

Comment:
This gives me `TypeError: errors.communities.map is not a function` if I press `Upload` without entering any values (as it is a string, not a list of strings)

Reply:
In that case the type definition for `errors.communities` was wrong and should be looked at. I just made it pass typechecks here.

Reply:
Yeah that's why I left it at the time. I think we might need to make an issue on react-hook-form. It's either a list of strings or a string

Reply:
`PackageUploadFormInputs.communities` should be changed to `NestedValue<string[]>` as "non-primitive types" should be wrapped with `NestedValue` (the docs aren't very clear). This fixes the type for `errors`